Add String Padding + FileOrder Requirements, Dependencies Upgrade, Add Tests

dreamsyntax opened the pull request:
Why:
* Now matches ShadowMTP
* File order matters for entries, due to hardcoded mapping
* Tests previously did not compare content

Change:
* Modify import/export to only use files specified from FileOrder.txt and to use them in order
* Fix string padding (weird requirement for in-game parser)
* Identify add test for file order
* Update dependencies: upgrade to .NET 5, .NETStandard 2.1, Reloaded library update
* Add export test for file content
* Add two possible cases in MTP types

dreamsyntax committed:
update dependencies, upgrade to .NET 5, .NETStandard 2.1, add SUPERSHADOW asset, add true export tests, identify bug with ToMtp
fix string padding bug, identify file order matters for entries, add test for file order
modify import/export to only use files specified from FileOrder.txt and to use them in order
override ToString to list fileName of MTN
conform to sewer style

Sewer56 commented:
LGTM, just a few minor things.

- In some places; the style don't match the rest of the codebase..

Examples: (brackets) 

https://github.com/Sewer56/ShadowMTPSharp/blob/d938f7cd38fc083622ff2403e79fb170048d1dfd/MTPLib/MotionPackage.cs#L153-L159
https://github.com/Sewer56/ShadowMTPSharp/blob/d938f7cd38fc083622ff2403e79fb170048d1dfd/MTPLibTest/Exporting.cs#L49-L59

- It seems that in some files; you/your text editor might changed the line ending encoding; making things appear as if the whole file was modified.
  - It makes the commit history look unclean; so you should look into it :P

dreamsyntax commented:
You can disable whitespace/newline filtering in GitHub diffs.
I left it in windows format -> AUTOCRLF was false, changed.

As for the styling, seeing as this is not actively maintained I'm leaving it (not really).

Updated and squashed
